into/backends/csv.py

```diff
@@ -257,6 +250,16 @@ def convert_dataframes_to_temporary_csv(data, **kwargs):
     return append(csv, data, **kwargs)
 
 
+@append.register(CSV, Temp(CSV))
+def temp_csv_to_csv(a, b, **kwargs):
+    return append(a, CSV(b.path), **kwargs)
+
+
+@convert.register(Temp(CSV), CSV)
+def csv_to_temp_csv(csv, **kwargs):
+    return Temp(CSV)(csv.path, **kwargs)
```

Comment:
Beware, this might cause the original file to be lost.  Probably we want a new filename and an os-level copy.